### PR TITLE
3.0 updates

### PR DESCRIPTION
**Pull request by bicknellr**
- Unstable auto-formatting.
- Async demo doesn't work on Firefox: the name specifier in the dynamic import isn't being rewritten.

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -11,35 +11,27 @@
...
-
-document.body.appendChild($_documentContainer.content);
-</script>
-
   <script type="module">
-const $_documentContainer = document.createElement('template');
+    import '@polymer/iron-demo-helpers/demo-pages-shared-styles.js';
+
+    const template = document.createElement('template');
+    template.innerHTML = `
```
Comment (aomarks) [inline demo/index.html]: does it not work if you just put this HTML in the document like it originally was?

Comment (e111077) [reply demo/index.html]: I agree with Al, I think this is the best solution for now. It looks like clang-format is shitting itself with template literals

Comment (bicknellr) [reply demo/index.html]: Same issue as [this comment I left on iron-fit-behavior](https://github.com/PolymerElements/iron-fit-behavior/pull/93#discussion_r199027618). :/ I'll ask @azakus about this tomorrow.

**Review by aomarks — APPROVED** [review]

**Review by dfreedm — COMMENTED** [review]
Travis changes LGTM

**Review by e111077 — APPROVED** [review]

**Pull request merged**